### DISCUSSION TITLE
Logo similar to Slack's

### DISCUSSION THREAD

**Discussion by sno2** [AppFlowy-IO/AppFlowy#111] [Ideas]
I dunno if this is on purpose or not but your logo is extremely similar to Slack's.  Here's a comparison:

![image](https://user-images.githubusercontent.com/43641633/142752568-dc3b4d0f-9d5f-4156-896a-9cfe924af471.png)

![image](https://user-images.githubusercontent.com/43641633/142752570-bcac53ff-ef0d-40ad-88ea-b560263fcbd9.png)

As you can see, the colors basically match in the four corners except the top left.  Furthermore, it basically has a similar shape patterns in the corners.

Personally, I think ya'll should change your logo to more unique as I had thought it was the Slack logo and that this was a project by Slack initially.

**Comment by annieappflowy** [comment]
I appreciate your point of view. Will leave it open here. Welcome feedback 😁

**Comment by SleepDaemon** [comment]
ig make it a flowing wave since its called flowy

**Comment by notd5a-alt** [comment]
I think a wave would be great as well

**Comment by branhillsdesign** [comment]
![appflowy io logo](https://user-images.githubusercontent.com/51428934/146666588-61eaca56-a955-424a-80b3-b2c19dc1b6df.png)

Here's an initial illustration I've come up with as a concept @sno2. Since at the core, as of now, appflowy is a note taking app; using a quill pen and combining that with a wave for "flow". The logo certainly needs more refinement, but I think this is a good starting place.

**Reply by sno2** [reply]
Wow, that's really good. However, I am not actually on the appflowy team.

**Reply by rileyhawk1417** [reply]
Any opinions or suggestions are welcome @sno2 😄

**Comment by astrolemonade** [comment]
I am not in the appflowy team but having a more recognizable icon would be more helpful. The one you showed looks good but it gives me Microsoft Edge vibes.

**Reply by MikeWallaceDev** [reply]
My personal opinion for a logo : no blue.  It's everywhere.
I like how the logo is dynamic, lots of movement in it. Very cool.

Also, I'm dumb. I didn't that it looked like a feather. Ohhhhhhhhhhh a quill. Nice. But will most people get that?
The concept of "writting mechanism" and "wave" is genius. 

I can see that thing being animated, unravelling and turning into an animated character. (Hi! I'm Flowy! I see that you are writing a document, can I help you with that?) :)

Will it scale down well? (think favicon)

**Reply by SleepDaemon** [reply]
I thought it was a feather rather than quill, second Blue is a pleasing color, and if we are going to go with a "Wave" then the color is got to be Blue.

**Comment by SleepDaemon** [comment]
Here is some "Wave" logos for reference.

![image](https://user-images.githubusercontent.com/88533953/146894126-f3ca860b-6f52-4374-b3d7-dd330cda3814.png)
![image](https://user-images.githubusercontent.com/88533953/146894219-5e4ef8c9-ecd4-46f8-ac0e-a98e6177a548.png)
![image](https://user-images.githubusercontent.com/88533953/146894230-103fab95-b419-4ff7-b256-7f9035ef0043.png)
![image](https://user-images.githubusercontent.com/88533953/146894243-445ae88b-5518-4a7a-9d5c-fe83812d09b6.png)

**Reply by SleepDaemon** [reply]
and yes i did just google "wave logo" ill try to make one myself in illustrator, a basic concept one. But i aignt that good so hopefully someone with more skill and experience can make it.

**Comment by branhillsdesign** [comment]
<img width="433" alt="appflowy logo" src="https://user-images.githubusercontent.com/51428934/146954331-761b98df-5435-449d-9e14-30ef5b38347c.png">

**Reply by branhillsdesign** [reply]
Changed up the wave to read more like a feather. Added the typographic mark, then made sure it looks good in one color along with on a light and dark background.

**Comment by MikeWallaceDev** [comment]
I always thought that it looked like a feather. I just didn't make the jump to "quill". I like it more and more :-)
